src/app/routes/utils/pageTypes.js

```diff
@@ -1,4 +1,5 @@
 export const ARTICLE_PAGE = 'article';
+export const MEDIA_ARTICLE_PAGE = 'mediaArticlePage';
```

Comment:
Probably nitpicky but should this just be called `mediaArticle`? I think `frontPage` is the only page type with the word `page` in it, so for things like analytics it may be better to just keep it as `mediaArticle`.